modules/swift/examples/code_snippets/SampleCodeTest.swift

```diff
@@ -58,17 +58,28 @@ class SampleCodeTest {
         // end::logging[]
     }
 
+    func dontTestConsoleLogging() throws {
+        // tag::console-logging[]
+        Database.log.Console.domains = LogDomains.all // <.>
```

Comment:
do we need to specify explicitly the `LogDomains.all`? also console is lowercase. 
```suggestion
        Database.log.console.domains = .all // <.>
```